### PR TITLE
Added the LSF to the list

### PR DESCRIPTION
**Pull request by tiatmed**
Added the Lanka Software Foundation's JSON file

### CODE REVIEW

File: directory.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
 	"LR": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Russia/LinuxRussia-api.json",
 	"LUGM": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/India/Manipal.json",
 	"LINUXJP":"https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Japan/LinuxJapan-api.json",
+	"LSF": "https://raw.githubusercontent.com/tiatmed/fossasia-communities/master/Sri%20Lanka/LankaSoftwareFoundation-api.json"
```
Comment (samarjeet27) [inline directory.json]: A comma here for valid JSON

**Comment by tiatmed** [comment]
I have made the change.

**Comment by samarjeet27** [comment]
@tiatmed , it's recommended that you use the fossasia/fossasia-communities link instead of the tiatmed/fossasia-communities link

**Comment by ty221** [comment]
@samarjeet27 I have just merged his pull request in fossasia-communites. @tiatmed please update this PR :D

**Pull request closed without merging**